installation/aws-container.md

```diff
@@ -1,45 +1,57 @@
 # Containers on AWS
 
-AWS maintains a distribution of Fluent Bit combining the latest official release with a set of Go Plugins for sending logs to AWS services. AWS and Fluent Bit are working together to rewrite their plugins for inclusion in the official Fluent Bit distribution.
+AWS maintains a distribution of Fluent Bit combining the latest official release with
+a set of Go Plugins for sending logs to AWS services. AWS and Fluent Bit are working
+together to rewrite their plugins for inclusion in the official Fluent Bit
+distribution.
 
 ## Plugins
 
-Currently, the [AWS for Fluent Bit](https://github.com/aws/aws-for-fluent-bit) image contains Go Plugins for:
+The [AWS for Fluent Bit](https://github.com/aws/aws-for-fluent-bit) image contains Go
+Plugins for:
 
-* [Amazon CloudWatch Logs](https://github.com/aws/amazon-cloudwatch-logs-for-fluent-bit)
-* [Amazon Kinesis Firehose](https://github.com/aws/amazon-kinesis-firehose-for-fluent-bit)
-* [Amazon Kinesis Streams](https://github.com/aws/amazon-kinesis-streams-for-fluent-bit)
+- Amazon CloudWatch as  `cloudwatch_logs`. See the
+  [Fluent docs](https://docs.fluentbit.io/manual/pipeline/outputs/cloudwatch) or the
```

Comment:
Should these be "Fluent Bit docs" and not "Fluent docs"? Also, these are absolute links. Is that how the links are being handled in the docs?

Reply:
they mostly seem to be, and I haven't wanted to spend the time struggling with the (working) link structure when I could be fixing things that are actually broken or wrong.

Fighting with the links is a future problem.

Reply:
and yes, I fixed them to say "Fluent Bit docs"